Bruteforce_parameter_sweep.ipynb

```diff
@@ -240,20 +240,7 @@
     "file = de.Make_dir('test_1atm_20000Hz_2D')\n",
     "\n",
     "# save all settings (full_bubble_model.py, parameters.py, ranges) as txt:\n",
-    "combined_str = f'''parameters settings:\n",
```

Comment:
This information (parameter settings: [...] and full_bubble_model settings: [...]) are no longer saved redundantly in `settings.txt`